pkg/webhook/mutation/pod/v2/common/volumes.go

```diff
@@ -0,0 +1,94 @@
+package common
```

Comment:
> pkg/webhook/mutation/pod/common/...: The common logic, that can be used both in v1 and v2

why do we have another common inside v2 `pkg/webhook/mutation/pod/v2/common/volumes.go`?

Reply:
the fact that there are only volumes here doesn't make it necessarily a volumes pacakge :D

I will move another helper lib in here, because adding everything to `kubeobjects` is not maintainable

Reply:
fixed it in https://github.com/Dynatrace/dynatrace-operator/pull/4576/commits/990bdd34bcc2ee7f90e0a77d993d9644fc43d882 (hopefully it makes more sense now)